### DISCUSSION TITLE
Recommended Hardware/Software

### DISCUSSION THREAD

**Discussion by ProfeSergioM** [nikosT/Gisola#43] [Q&A]
Hi! i have the possibility to test GISOLA in a fresh server here in my office, but I need to tell about the specs of this one

Do you have some recommendations in terms of OS and hardware?

Saludos from Chile!

**Comment by nikosT** [comment]
Hi @ProfeSergioM ,

In terms of OS a typical Linux distribution is enough, e.g. Ubuntu.
For hardware, a multi-core CPU is suggested for faster parallel processing. GPU is optional and only parallelizes the inversion procedures (not the seismological stream corrections). GPU must be able to perform double-precision operations fast to be useful (e.g. Nvidia V100) -quite expensive-. It's not tested on other Ventor GPU cards apart from Nvidia. I believe it's better to have a good CPU with many cores available to parallelize tasks. Also, a RAM of at least 16 GB (not a minimum requirement though) and SSD disks help.

I hope this helps you, please feel free to ask anything else. And let us know your experience with the software.
